HTML-CSS/Tasarım ödevi/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tasarım</title>
+</head>
+<body>
+    <div class="grid">
+        <div id="box-1"></div>
+        <div id="box-2"></div>
+        <div id="box-3"></div>
+        <div id="box-4"></div>
+        <div id="box-5"></div>
+        <div id="box-6"></div>
+        <div id="box-7"></div>
+        <div id="box-8"></div>
+        <div id="box-9"></div>
```

Comment:
BEM Metadolojisini kullanmamışsın. Çoğu zaman BEM (Block Element Modifier) metadolojisini kullanmak, kodu tekrar analiz ederken de CSS yazarken de yararına olacaktır. Elbette her durumda BEM kullanmak doğru olmayabilir. Örneğin, helper class'lar bunlardan birisidir.

BEM Metadolojisi: https://en.bem.info/methodology/quick-start/
Helper Class'lar hakkında detaylı bilgi edinmek için makale: https://blog.devgenius.io/how-writing-css-helper-classes-improved-my-productivity-c2360d0dab78